src/main/java/uk/gov/hmcts/reform/professionalapi/controller/constants/ProfessionalApiConstants.java

```diff
@@ -75,10 +75,12 @@ private ProfessionalApiConstants() {
     public static final String ERROR_MSG_PBA_INVALID_FORMAT =
             "PBA numbers must start with PBA/pba and be followed by 7 alphanumeric characters";
     public static final String ERROR_MSG_PBAS_ENTERED_ARE_INVALID = ". The following PBAs entered are invalid: ";
+    public static final String ERROR_MSG_CONTACT_INFO_IS_MISSING = " is missing";
```

Comment:
Can we rename this constant to be `IS_MISSING` to make it more generic and reusable